.github/labeler.yml

```
@@ -0,0 +1,76 @@
+# Configuration for GitHub labeler action
+
+# Component changes
+components:
+  - changed-files:
+    - any-glob-to-any-file: 'src/components/**/*'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)